WpfAnimatedGif/WpfAnimatedGif.csproj

```diff
@@ -1,73 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D129789C-3096-4D0B-8DD7-FE24A4DF4B21}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>WpfAnimatedGif</RootNamespace>
-    <AssemblyName>WpfAnimatedGif</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile>Client</TargetFrameworkProfile>
+    <TargetFrameworks>net35;net4;net45;net46;net472;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Is it actually necessary to include all those frameworks?
For instance, net40 would be enough to support every net4* framework, and a netcoreapp3.1 app can use a netcoreapp3.0 assembly.
I think keeping just net35, net40 and netcoreapp3.0 is enough

Reply:
No not really, i was just tinkering with the versions and ended up with that lol. So keeping net35, net40 and netcoreapp3.0 should be enough.